docs/src/main/tut/datatypes/eithert.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: docs
+title:  "EitherT"
+section: "data"
+source: "core/src/main/scala/cats/data/EitherT.scala"
+scaladoc: "#cats.data.EitherT"
+---
+# EitherT
+
+`Either` can be used for error handling in most situations. However, when
+`Either` is placed into effectful types such as `Option` or`Future`, a large
+amount of boilerplate is required to handle errors. For example, consider the
+following program:
+
+```tut:book
+import scala.util.Try
+import cats.syntax.either._
```

Comment:
We decide to use the uber import `import cats.implicits._` in the documentation, see issue #1026.